addons/stock/wizard/stock_picking_return.py

```diff
@@ -208,7 +210,7 @@ def action_create_returns_all(self):
         self.ensure_one()
         for return_move in self.product_return_moves:
             stock_move = return_move.move_id
-            if not stock_move or stock_move.state == 'cancel' or stock_move.scrapped:
```

Comment:
I'm not sure I understand this change either. How can you return a move that was cancelled or scrapped?

Reply:
my bad